components/Common/Breadcrumbs/BreadcrumbRoot/index.tsx

```diff
@@ -0,0 +1,57 @@
+import HomeIcon from '@heroicons/react/24/outline/HomeIcon';
+import type { FC, PropsWithChildren } from 'react';
+import { useIntl } from 'react-intl';
+
+import BreadcrumbItem from '@/components/Common/Breadcrumbs/BreadcrumbItem';
+import LocalizedLink from '@/components/LocalizedLink';
+
+import styles from './index.module.css';
+
+type BreadcrumbRootProps = {
+  hideHome?: boolean;
+};
+
+const BreadcrumbRoot: FC<PropsWithChildren<BreadcrumbRootProps>> = ({
+  hideHome = false,
+  children,
+}) => {
+  const { formatMessage } = useIntl();
+
+  const navigateToHome = formatMessage({
+    id: 'components.common.breadcrumbs.navigateToHome',
+  });
+
+  return (
+    <nav aria-label="breadcrumb">
+      <ol
+        itemScope
+        itemType="https://schema.org/BreadcrumbList"
+        className={styles.list}
+      >
+        {!hideHome && (
+          <BreadcrumbItem>
+            <LocalizedLink
+              itemScope
+              itemType="http://schema.org/Thing"
+              itemProp="item"
+              itemID="/"
+              href="/"
+            >
+              <span itemProp="name">
```

Comment:
Does it need to be surrounded by a span? Quite sure you could just add itemProp to the SVG.

Reply:
I had to do this ugly hack

```diff
<HomeIcon
+ {...{ itemProp: 'name' as ComponentProps<'svg'> }}
- itemProp="name"
  ...
/>
```

else I am getting compiler error

```
Type '{ itemProp: string; title: string; "aria-label": string; className: string; }' is not assignable to type 'IntrinsicAttributes & Omit<SVGProps<SVGSVGElement>, "ref"> & { title?: string | undefined; titleId?: string | undefined; } & RefAttributes<...>'.
  Property 'itemProp' does not exist on type 'IntrinsicAttributes & Omit<SVGProps<SVGSVGElement>, "ref"> & { title?: string | undefined; titleId?: string | undefined; } & RefAttributes<...>'.ts(2322)
```

Not quite sure why `HomeIcon` type does not support adding `itemProp`.